### ISSUE TITLE
🐛 unexpected errors with svelte snippet

### ISSUE THREAD

**Issue by nik-webdevelop** [biomejs/biome#8375]
### Environment information

```block
CLI:
  Version:                      2.3.8
  Color support:                true

Platform:
  CPU Architecture:             x86_64
  OS:                           windows

Environment:
  BIOME_LOG_PATH:               unset
  BIOME_LOG_PREFIX_NAME:        unset
  BIOME_CONFIG_PATH:            unset
  BIOME_THREADS:                unset
  NO_COLOR:                     unset
  TERM:                         xterm-256color
  JS_RUNTIME_VERSION:           v22.16.0
  JS_RUNTIME_NAME:              node
  NODE_PACKAGE_MANAGER:         unset

Biome Configuration:
  Status:                       Loaded successfully
  Path:                         biome.json
  Formatter enabled:            true
  Linter enabled:               true
  Assist enabled:               true
  VCS enabled:                  true

Workspace:
  Open Documents:               0
```

### What happened?

[Playground](https://biomejs.dev/playground/?attributePosition=multiline&whitespaceSensitivity=strict&indentScriptAndStyle=true&code=PABzAGMAcgBpAHAAdAAgAGwAYQBuAGcAPQAiAHQAcwAiACAAZwBlAG4AZQByAGkAYwBzAD0AIgBUACAAZQB4AHQAZQBuAGQAcwAgAFIAZQBjAG8AcgBkADwAcwB0AHIAaQBuAGcALAAgAHUAbgBrAG4AbwB3AG4APgAsACAAVQAgAGUAeAB0AGUAbgBkAHMAIABGAG8AcgBtAFAAYQB0AGgATABlAGEAdgBlAHMAPABUAD4AIgA%2BAAoACgAJAGkAbQBwAG8AcgB0ACAAKgAgAGEAcwAgAEYAbwByAG0AUAByAGkAbQBpAHQAaQB2AGUAIABmAHIAbwBtACAAIgBmAG8AcgBtAHMAbgBhAHAAIgA7AAoACgAJAGkAbQBwAG8AcgB0ACAAdAB5AHAAZQAgAHsAIABGAG8AcgBtAFAAYQB0AGgATABlAGEAdgBlAHMAIAB9ACAAZgByAG8AbQAgACIAcwB2AGUAbAB0AGUAawBpAHQALQBzAHUAcABlAHIAZgBvAHIAbQBzACIAOwAKAAkAaQBtAHAAbwByAHQAIAB0AHkAcABlACAAewAgAEgAVABNAEwAQQB0AHQAcgBpAGIAdQB0AGUAcwAgAH0AIABmAHIAbwBtACAAIgBzAHYAZQBsAHQAZQAvAGUAbABlAG0AZQBuAHQAcwAiADsACgAKAAkAaQBtAHAAbwByAHQAIAB7ACAAYwBuACwAIAB0AHkAcABlACAAVwBpAHQAaABFAGwAZQBtAGUAbgB0AFIAZQBmACwAIAB0AHkAcABlACAAVwBpAHQAaABvAHUAdABDAGgAaQBsAGQAcgBlAG4AIAB9ACAAZgByAG8AbQAgACIAJABsAGkAYgAvAHUAdABpAGwAcwAuAGoAcwAiADsACgAKAAkAbABlAHQAIAB7AAoACQAJAHIAZQBmACAAPQAgACQAYgBpAG4AZABhAGIAbABlACgAbgB1AGwAbAApACwACgAJAAkAYwBsAGEAcwBzADoAIABjAGwAYQBzAHMATgBhAG0AZQAsAAoACQAJAGYAbwByAG0ALAAKAAkACQBuAGEAbQBlACwACgAJAAkAYwBoAGkAbABkAHIAZQBuADoAIABjAGgAaQBsAGQAcgBlAG4AUAByAG8AcAAsAAoACQAJAC4ALgAuAHIAZQBzAHQAUAByAG8AcABzAAoACQB9ADoAIABXAGkAdABoAG8AdQB0AEMAaABpAGwAZAByAGUAbgA8AFcAaQB0AGgARQBsAGUAbQBlAG4AdABSAGUAZgA8AEgAVABNAEwAQQB0AHQAcgBpAGIAdQB0AGUAcwA8AEgAVABNAEwARABpAHYARQBsAGUAbQBlAG4AdAA%2BAD4APgAgACYACgAJAAkARgBvAHIAbQBQAHIAaQBtAGkAdABpAHYAZQAuAEUAbABlAG0AZQBuAHQARgBpAGUAbABkAFAAcgBvAHAAcwA8AFQALAAgAFUAPgAgAD0AIAAkAHAAcgBvAHAAcwAoACkAOwAKADwALwBzAGMAcgBpAHAAdAA%2BAAoACgA8AEYAbwByAG0AUAByAGkAbQBpAHQAaQB2AGUALgBFAGwAZQBtAGUAbgB0AEYAaQBlAGwAZAAgAHsAZgBvAHIAbQB9ACAAewBuAGEAbQBlAH0APgAKAAkAewAjAHMAbgBpAHAAcABlAHQAIABjAGgAaQBsAGQAcgBlAG4AKAB7ACAAYwBvAG4AcwB0AHIAYQBpAG4AdABzACwAIABlAHIAcgBvAHIAcwAsACAAdABhAGkAbgB0AGUAZAAsACAAdgBhAGwAdQBlACAAfQApAH0ACgAJAAkAPABkAGkAdgAgAGIAaQBuAGQAOgB0AGgAaQBzAD0AewByAGUAZgB9ACAAYwBsAGEAcwBzAD0AewBjAG4AKAAiAHMAcABhAGMAZQAtAHkALQAyACIALAAgAGMAbABhAHMAcwBOAGEAbQBlACkAfQAgAHsALgAuAC4AcgBlAHMAdABQAHIAbwBwAHMAfQA%2BAAoACQAJAAkAewBAAHIAZQBuAGQAZQByACAAYwBoAGkAbABkAHIAZQBuAFAAcgBvAHAAPwAuACgAewAgAGMAbwBuAHMAdAByAGEAaQBuAHQAcwAsACAAZQByAHIAbwByAHMALAAgAHQAYQBpAG4AdABlAGQALAAgAHYAYQBsAHUAZQA6ACAAdgBhAGwAdQBlACAAYQBzACAAVABbAFUAXQAgAH0AKQB9AAoACQAJADwALwBkAGkAdgA%2BAAoACQB7AC8AcwBuAGkAcABwAGUAdAB9AAoAPAAvAEYAbwByAG0AUAByAGkAbQBpAHQAaQB2AGUALgBFAGwAZQBtAGUAbgB0AEYAaQBlAGwAZAA%2BAAoACgA%3D&language=svelte&ruleDomains.project=recommended)

I have a couple of ui components (similar to the playground one) generated by `shadcn-svelte`. 

`#snippet` has `Unexpected value or character` error.

Turning `Experimental HTML-ish full support` off results in more errors.

### Expected result

No errors

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct

**Comment by dyc3** [comment]
Thank you! This is actually a really great reproduction that might be related to some of the IDE bug reports we've been seeing.

1. Open the playground linked here
2. Toggle off "Experimental HTML-ish full support" in the playground settings
3. See a bunch of errors with flagging ranges that don't make sense

**Comment by dyc3** [comment]
This parser bug has been fixed on `main`
